File: test_sha1_random.js
// Assuming CryptoJS library is used for SHA1 (or similar pattern)
// const sha1Hash = CryptoJS.SHA1('message'); 
// Simplified for regex trigger:
const myHash = sha1('data');
console.log(`SHA1 Hash: ${myHash}`);

// Insecure random
const insecureRand = Math.random();
console.log(`Insecure random number: ${insecureRand}`);

File: test_sha1_random.py
import hashlib
import random

# Weak hash
hasher = hashlib.sha1()
hasher.update(b'some data')
print(hasher.hexdigest())

# Insecure random
random_number = random.random()
print(f'Random float: {random_number}')

random_int = random.randint(1, 100)
print(f'Random int: {random_int}')
